app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] I got an error when running the seed. This is because here on line 10 you are writing `comments_counter` while in your schema you have `CommentsCounter`. Kindly fix this

Reply:
This has been resolved.